undertow/api/src/main/java/org/wildfly/swarm/undertow/UndertowFraction.java

```diff
@@ -134,6 +172,9 @@ public void initialize(Fraction.InitContext initContext) {
         initContext.socketBinding(
                 new SocketBinding("https")
                         .port(SwarmProperties.propertyVar(UndertowProperties.HTTPS_PORT, "8443")));
+        initContext.socketBinding(
+                new SocketBinding("ajp")
+                        .port(SwarmProperties.propertyVar(UndertowProperties.AJP_PORT, "8009")));
```

Comment:
Should this call to create a SocketBinding be wrapped in a check for this.enableAJP?

Otherwise we're creating a SocketBinding for no reason, right?